Competition2017/src/competition/subsystems/shooter_belt/commands/RunBeltTracerSpeedModeCommand.java

```diff
@@ -0,0 +1,27 @@
+package competition.subsystems.shooter_belt.commands;
+
+import competition.subsystems.hybrid.commands.BaseShooterBeltAndWheelCommand;
+import competition.subsystems.shooter_belt.ShooterBeltSubsystem;
+import competition.subsystems.shooter_wheel.ShooterWheelSubsystem;
+
+public class RunBeltTracerSpeedModeCommand extends BaseShooterBeltAndWheelCommand {
+    
+    public RunBeltTracerSpeedModeCommand(ShooterBeltSubsystem shooterBeltSubsystem, ShooterWheelSubsystem shooterWheelSubsystem) {
+        super(shooterBeltSubsystem, shooterWheelSubsystem);
+        this.requires(shooterBeltSubsystem);
+    }
+    
+    @Override
+    public void initialize() {
+        log.info("Initializing");
+    }
+
+    @Override
+    public void execute() {
+        if(shooterWheelSubsystem.isAtSpeed()){
+            shooterBeltSubsystem.intakeUsingTracerSpeed();
```

Comment:
I would advocate for using Power instead of Speed in the short-run since speed isn't currently tuned/verified on the comp bot and if power works that's much simpler to verify this concept.